### DISCUSSION TITLE
QuickLook

### DISCUSSION THREAD

**Discussion by talmacsi** [derceg/explorerplusplus#498] [Ideas]
I would never use another file browser, if there would be a possibility to use QuickLook.
Please, help me!

**Comment by DejayRezme** [comment]
I wish for quicklook too, it's so amazing. 

Alternatively, in the browser I use an addon called imagus that creates a pop up for thumbnails and links to show the full images. Maybe this could be implemented in an explorer too - maybe only when hovering over the micro-icon in details view.
